packages/core/upload/admin/src/components/EditAssetDialog/index.js

```diff
@@ -116,6 +123,29 @@ export const EditAssetDialog = ({
     }
   };
 
+  if (folderStructureIsLoading) {
+    return (
+      <ModalLayout onClose={() => handleClose()} labelledBy="title">
+        <DialogHeader />
+        <LoadingBody minHeight="60vh" justifyContent="center" paddingTop={4} paddingBottom={4}>
+          <Loader>
+            {formatMessage({
+              id: getTrad('list.asset.load'),
+              defaultMessage: 'We are loading the page. This action can take a few seconds.',
```

Comment:
We finally decided on:

```suggestion
                       defaultMessage: 'Content is loading.',
```